build-ps/debian/percona-xtradb-cluster-server-5.7.preinst

```diff
@@ -152,6 +152,24 @@ case "$1" in
 
 	#DEBHELPER#
 	server_stop
+	currentver=$(dpkg -l | grep percona-xtradb-cluster-server-5.7 | awk '{print $3}' | awk -F'-' '{print $1}')
+	requiredver="5.7.19"
+	if [[ $currentver < $requiredver ]]; then
+cat >/usr/share/mysql/pxc_cluster_view.sql << EOL
```

Comment:
Why is the file in a heredoc?   Can't we just commit the file into the repo and install it with the package?